python/GafferArnoldUI/ArnoldRenderUI.py

```diff
@@ -96,6 +97,19 @@
 
 		],
 
+		"command" : [
+
+			"description",
+			"""
+			The system command used to invoke the renderer - this
+			can be edited to add any custom flags that are necessary,
+			or to use a different renderer. The ass filename is
```

Comment:
Maybe remove "or to use a different renderer"? That makes some sense on the RenderManRender node because several renderers implement the same "standard". But that's not really the case for Arnold.